scripts/rollup/plugins.mjs

```diff
@@ -49,6 +49,7 @@ export const makePlugins = () => [
         declaration: true,
         declarationDir: settings.types,
         target: 'esnext',
+        paths: process.env.USE_PATHS ? undefined : [],
```

Comment:
Because we split our build into three workers this way we can leave the paths intact and use the packages as intended on publish